### DISCUSSION TITLE
Track domains

### DISCUSSION THREAD

**Discussion by diegolr62** [postalserver/postal#2038] [Help with using Postal] [answered]
I surrender, I have tried all combinations. Here is what I have in DNS

track.example.com is CNAME of  mx.example.com

In postal.yml I have:

track_domain: mx.example.com

But the automatic checking when saving the tracking domaind says that is not correct.

What is wrong? Please your help

Regards

**Comment by willpower232** [comment, marked as answer]
> But the automatic checking when saving the tracking domaind says that is not correct.

The tracking domain you add should be different, i.e. track.clientdomain.com is CNAME of track.example.com

**Reply by diegolr62** [reply]
My mistake. The CNAME was mispelled.

**Reply by willpower232** [reply]
aha happens to us all :sweat_smile:
